site/src/examples/link/Variant.tsx

```diff
@@ -0,0 +1,13 @@
+import { ReactElement } from "react";
+import { Link, StackLayout } from "@salt-ds/core";
+
+export const Variant = (): ReactElement => (
+  <StackLayout>
+    <Link href="/" variant="primary">
```

Comment:
```suggestion
    <Link href="#" variant="primary">
```
 Reduce likelihood of the link appearing in visited state when someone visits this component page.